src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -1,508 +1,32 @@
-import React, { useEffect, useState } from 'react';
```

Comment:
The diff here is really big, but this component basically became the overview component in this directory. The index should probably house the routing logic, sub-routes, etc.